### PR TITLE
Make directives language agnostic

### PR DESCRIPTION
**Pull request by hamelsmu**
1. Make directives langauge agnostic where possible
2. Ignore directives that are python specific for non python cells/notebooks
3. Raise error if you try to `showdoc` or `export` non-python
4. Only test notebooks where kernel is python

I've tested an APL notebook with directives as well for rendering and it does work: notice the cell is hidden properly

imageimage

### CODE REVIEW

File: nbprocess/clean.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 from fastcore.script import *
 from fastcore.basics import *
 from fastcore.imports import *
+from fastcore.test import *
```
Comment (jph00) [inline nbprocess/clean.py:13]: this shouldn't be exported should it?

File: nbprocess/merge.py [hunk 2]
```diff
@@ -12,6 +12,7 @@
 from execnb.nbio import *
 from fastcore.script import *
 from fastcore import shutil
+from fastcore.test import *
```
Comment (jph00) [inline nbprocess/merge.py:15]: also here

File: nbprocess/clean.py (hunk 1 above)
Comment (hamelsmu) [reply nbprocess/clean.py]: Yes you are correct.  I missed this 🙇🏽

File: nbprocess/merge.py (hunk 2 above)
Comment (hamelsmu) [reply nbprocess/merge.py]: Fixed this

File: nbprocess/sync.py [hunk 3]
```diff
@@ -13,6 +13,7 @@
 from execnb.nbio import *
 from fastcore.script import *
 from fastcore.xtras import *
+from fastcore.test import *
```
[the remark this answers is not in the document]
Comment (hamelsmu) [reply nbprocess/sync.py]: This is done

File: nbprocess/processors.py [hunk 4]
```diff
@@ -71,10 +76,7 @@ def lang_identify(cell):
     "A preprocessor to identify bash/js/etc cells and mark them appropriately"
     if cell.cell_type == 'code':
         lang = _lang_pattern.findall(cell.source)
-        if lang:
-            lang = lang[0]
-            if lang=='js': lang='javascript'  # abbrev provided by jupyter
-            cell.metadata.language = lang
+        if lang: cell.metadata.language = lang[0]
```
[the remark this answers is not in the document]
Comment (hamelsmu) [reply nbprocess/processors.py:79]: Yes you are right, I added this to the dict in https://github.com/fastai/nbprocess/pull/98/commits/b56a23aabf9a97f2693f10ea92db9c90662af70a

**Comment by hamelsmu** [comment]
I am going to merge this to help with #100 (the fastcore imports)

**Pull request merged**